index.bs

```diff
@@ -249,22 +249,68 @@ interface NeuralNetworkContext {
 };
 </script>
 
-### add ### {#api-neuralnetworkcontext-add}
-Compute the element-wise binary addition of the two input tensors.
+### element-wise binary operations ### {#api-neuralnetworkcontext-binary}
+Compute the element-wise binary addition, subtraction, multiplication, division,
+maximum and minimum of the two input tensors.
 <script type=idl>
 partial interface NeuralNetworkContext {
   Operand add(Operand a, Operand b);
+  Operand sub(Operand a, Operand b);
+  Operand mul(Operand a, Operand b);
+  Operand div(Operand a, Operand b);
+  Operand max(Operand a, Operand b);
+  Operand min(Operand a, Operand b);
 };
 </script>
-<div algorithm=add>
+<div algorithm=binary>
     **Arguments:**
         - *a*: an {{Operand}}. The first input tensor.
         - *b*: an {{Operand}}. The second input tensor.
 
     **Returns:** an {{Operand}}. The output tensor that contains the result of
-    element-wise binary addition of the two input tensors.
+    element-wise binary operation of the two input tensors.
+
+    The element-wise binary operation will be broadcasted according to
+    [[!numpy-broadcasting-rule]]. The rank of the output tensor is the maximum
+    rank of the input tensors. For each dimension of the output tensor, its size
+    is the maximum size along that dimension of the input tensors.
+</div>
+
+### element-wise unary operations ### {#api-neuralnetworkcontext-unary}
+Compute the element-wise unary operation for input tensor.
+<script type=idl>
+partial interface NeuralNetworkContext {
+  Operand abs(Operand x);
+  Operand ceil(Operand x);
+  Operand cos(Operand x);
+  Operand exp(Operand x);
+  Operand floor(Operand x);
+  Operand log(Operand x);
+  Operand neg(Operand x);
+  Operand sin(Operand x);
```

Comment:
Since we have sin and cos, should we have tan as well?

Reply:
Add `tan` in the new commit. Please take a look.